lib/active_admin/filters/active.rb

```diff
@@ -8,13 +8,21 @@ class Active
 
       def initialize(resource_class, params)
         @resource_class = resource_class
-        @params = params.respond_to?(:to_unsafe_h) ? params.to_unsafe_h : params
+        @params = normalize_params(params)
         @scope = humanize_scope
         @filters = build_filters
       end
 
       private
 
+      def normalize_params(params)
+        if params.kind_of?(HashWithIndifferentAccess)
```

Comment:
in rails 4.2 ActionController::Parameters inherited from HashWithIndifferentAccess, it even have to_unsafe_h method, but it works not that way like in rails5